web/cypress/e2e/onboarding-as-foreign-business.spec.ts

```diff
@@ -158,4 +158,49 @@ describe("Onboarding for all industries when out of state nexus business [featur
       cy.get('[data-testid="header-link-to-profile"]');
     });
   });
+
+  describe("Onboarding for all industries when out of state remote worker business [feature] [all] [group4]", () => {
+    describe("Desktop", () => {
+      beforeEach(() => {
+        cy.loginByCognitoApi();
+      });
+
+      it("Onboarding for Out of State Remote Worker", () => {
+        cy.url().should("include", "onboarding?page=1");
+        onOnboardingPageNexusBusiness.selectBusinessPersonaRadio("FOREIGN");
```

Comment:
pebble - the `onOnboardingPageNexusBusiness` should be a separate object... like `onOnboardingPageRemoteWorkerBusiness`

Reply:
would it make more sense to change the object to be more general and be onOnboardingPageForeignBusiness, like the starting and existing?  They are technically still under the same umbrella

Reply:
I agree. We have foreign businesses broken out in our code as `NexusBusiness`, `RemoteWorker` and `RemoteSeller`. I think it makes sense for us to be specific in the tests.

Reply:
Keeping it more general would mean that certain functions like selecting the industry would be available for remote, sellers and remote worker. I think it would make more sense to have specific objects.

Reply:
updated